src/arrayMethodSort.js

```diff
@@ -1,11 +1,40 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringB > stringA) {
+    return -1;
+  } else {
+    return 0;
+  }
```

Comment:
I think that else keyword in second condition and last block are not much needed here, cause you return values if condition is true (not sure though)
```suggestion
  if (stringA > stringB) {
    return 1;
  }
  
  if (stringB > stringA) {
    return -1;
  }
  
  return 0;
```

Reply:
Yep, Illia is correct. If you have `return` inside `if`, the following `else` is redundant.